jhipster/src/main/webapp/app/patient.tsx

```diff
@@ -181,15 +181,38 @@ const togglePosition = () => {
   setIsFixed((prevIsFixed) => !prevIsFixed);
 };
 
+const [activeTab, setActiveTab] = useState('accueil');
+
+  const changeTab = (tabName) => {
+    setActiveTab(tabName);
+  };
+
+  const renderTabContent = () => {
+    switch (activeTab) {
+      case 'note':
+        return <div>Notes content goes here</div>;
+      case 'rappel':
+        return <div>Rappel content goes here</div>;
+      case 'alerte':
+        return <div>Alerte content goes here</div>;
+      default:
+        return <div><Col md="12" className={`graphs ${isFixed ? 'fixed' : 'relative'}`}>
+        <Line options={options} data={data} />
+        <Line options={options1} data={data1} />
+         <Line options={options2} data={data2} />
+        </Col></div>;
+    }
+  };
+
   const patientEntity = useAppSelector(state => state.patient.entity);
   return (
-    <Row className="container">
+    <Row className="container-fluid">
```

Comment:
HTML code starts to be a bit too big, if it has to grow again, separate it in multiple files